packages/shared/src/components/MobileHeaderRankProgress.tsx

```diff
@@ -19,10 +19,15 @@ const Wrapper = ({ children, ...props }: WrapperProps) => {
   );
 };
 
-export default function MobileHeaderRankProgress(): ReactElement {
+interface MobileHeaderRankProgressProps {
+  sidebarRendered: boolean;
+}
+export default function MobileHeaderRankProgress({
+  sidebarRendered,
+}: MobileHeaderRankProgressProps): ReactElement {
   const { isLoading, rank, nextRank } = useReadingRank();
 
-  if (isLoading) {
+  if (isLoading || sidebarRendered) {
```

Comment:
If sidebar rendered is true shouldn't we just return an empty element? `<></>`

Reply:
Changed it to this now.
Initially I was afraid we would get the flicker back on mobile, but it seems to be fine.